runtime-components/non-repudiation-app/src/main/scala/com/daml/nonrepudiation/app/OptionParser.scala

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.nonrepudiation.app
+
+import java.net.InetSocketAddress
+
+private[app] object OptionParser extends scopt.OptionParser[Configuration](NonRepudiationApp.Name) {
+
+  head(NonRepudiationApp.Name)
+
+  opt[String]("ledger-host")
+    .required()
+    .action(setLedgerHost)
+    .text("The host address of the participant. Required.")
+
+  opt[String]("ledger-port")
+    .required()
+    .action(setLedgerPort)
+    .text("The port of the participant. Required.")
+
+  opt[Map[String, String]]("jdbc")
+    .required()
+    .action(setJdbcConfiguration)
+    .text(
+      """Contains comma-separated key-value pairs. Where:
+      |        url -- JDBC connection URL, beginning with jdbc:postgresql,
+      |        user -- user name for database user with permissions to create tables,
+      |        password -- password of database user,
+      |Example: "url=jdbc:postgresql://localhost:5432/nonrepudiation,user=nonrepudiation,password=secret"
+      |""".stripMargin
+    )
+
+  opt[String]("proxy-host")
+    .action(setProxyHost)
+    .text(
+      s"The interface over which the proxy will be exposed. Defaults to ${Configuration.Default.proxyAddress.getAddress}."
+    )
+
+  opt[String]("proxy-port")
+    .action(setProxyPort)
+    .text(
+      s"The port over which the proxy will be exposed. Defaults to ${Configuration.Default.proxyAddress.getPort}."
+    )
+
+  opt[String]("api-host")
+    .action(setApiHost)
+    .text(
+      s"The interface over which the non-repudiation API will be exposed. Defaults to ${Configuration.Default.proxyAddress.getAddress}."
+    )
+
+  opt[String]("api-port")
+    .action(setApiPort)
+    .text(
+      s"The port over which the non-repudiation API will be exposed. Defaults to ${Configuration.Default.proxyAddress.getPort}."
```

Comment:
Defaults seem off, this is referring to `proxyAddress` instead of `apiAddress`.

Reply:
Already fixed, thanks.